### PR TITLE
Fixes so we can register package

### PR DESCRIPTION
**Pull request by millerjoey**
Re.
https://github.com/JuliaRegistries/General/pull/29165#issuecomment-771261990

- [x] add compat for julia
- [x] Name change too? :'( 
- [x] Added TagBot.yml

### CODE REVIEW

File: Project.toml
Comment (ghost) [inline Project.toml]:
```suggestion
julia = "1.0"
```
There's nothing in here specific to later versions of julia. Our CI is run against julia 1.0, and i think we can support it! :)

**Review by ghost — APPROVED** [review]
Other than dropping down to julia 1.0, this LGTM. Thanks!

-----

Regarding the name change, I don't think we need to change the name. We can wait for some feedback from them, but it's only conflicting because it's a short word. I think it's a cute enough name that it's worth trying to keep it.

But if they don't love it, we could rename to:
- `MuttsInterface.jl`

What do you think about that? I actually kind of like it, since it's clearer about what this package provides?

**Comment by millerjoey** [comment]
Oh yeah, I like `MuttsInterface` better. You only have to type it a few times if you're using it and yes it's more clear. Your call though, keep or change?

**Pull request merged**
